### PR TITLE
WxQT: bump, debuginfo, versioncompat

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

**Comment by extrowerk** [comment]
I should disable OGL, right?

**Comment by waddlesplash** [comment]
LGTM

File: dev-qt/wxqt/wxqt-3.1.1.recipe [hunk 1]
```diff
@@ -120,7 +152,7 @@ PATCH()
 
 BUILD()
 {
-	export CXXFLAGS="-std=c++11"
+	export CXXFLAGS="-std=c++11 -fpermissive"
```
Comment (alaviss) [inline dev-qt/wxqt/wxqt-3.1.1.recipe]: I'm a bit skeptical about adding `-fpermissive`, as it allows non-conforming code to compile. If upstream doesn't do this by default, then I think the problem is in our patches.

I don't think explicitly setting c++11 is necessary.

Comment (waddlesplash) [reply dev-qt/wxqt/wxqt-3.1.1.recipe]: It was when we were on GCC5. And no, the Qt port is a bit experimental, so this is probably ok, I think?

**Pull request merged**